com.diy.software.test/src/com/diy/software/test/logic/ItemsControlTest.java

```diff
@@ -625,8 +625,13 @@ public class AttendantListenerStub implements AttendantControlListener {
     	public void attendantApprovedBags(AttendantControl ac) {
     		attendantBags = true;
     	}
-    	
-    	public boolean getAttendantBags() {
+
+		@Override
+		public void attendantPreventUse(AttendantControl ac) {
```

Comment:
the spacing in this file looks off

Reply:
I noticed that as well, but in my IDE it looks normal and if I try to commit there's no changes. So I'm not sure why it looks like that.

Reply:
I think it's just eclipse jiding what's actually happening code wise. I'll fix this at some point later down the line I guess